src/Plugin/QueueWorker/CronEncryptedFieldUpdate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\field_encrypt\Plugin\QueueWorker\CronEncryptedFieldUpdate.
+ */
+
+namespace Drupal\field_encrypt\Plugin\QueueWorker;
+
+use Drupal\field_encrypt\Plugin\QueueWorker\EncryptedFieldUpdateBase;
+
+/**
+ * A Queue Worker that updates field encryption on cron run.
```

Comment:
We should add to the readme about the proposed process for this. I'm OK with trying it and seeing if the community at large has other ideas